Dockerfile

```diff
@@ -38,7 +38,9 @@ RUN wget --progress=bar:force -P /tmp http://www.neuro.uni-jena.de/cat12/${CAT_F
  && mv /opt/${CAT_FULLVERSION}_${MATLAB_VERSION}_MCR_Linux /opt/spm \
  && /opt/spm/run_spm12.sh ${MCRROOT} --version \
  && chmod +x /opt/spm/spm12 /opt/spm/*.sh \
+ && chmod +x /opt/spm/spm12_mcr/home/gaser/gaser/spm/spm12/toolbox/cat12/CAT.glnx86/CAT_* \
  && rm -rf /tmp/*
+RUN cp /opt/spm/spm12_mcr/home/gaser/gaser/spm/spm12/toolbox/cat12/cat_long_main.txt /opt/spm/spm12_mcr/home/gaser/gaser/spm/spm12/toolbox/cat12/cat_long_main.m
```

Comment:
I don't like permanently replacing a compiled Matlab file by an existing script file (text). Would it be possible to rename it to something else such as `cat_long_main2.m`?

Reply:
Aaah.. they removed the accompanying `.txt` file here https://github.com/ChristianGaser/cat12/commit/febd62595be99f2bad049d9df93e920903fb3b06

So that line can be removed then as it shouldn't be necessary anymore for future builds. Or it could be changed to just deleting the `.txt` file if someone decides to use your Dockerfile to build a historic version (quite likely actually).

Reply:
As is not an issue anymore (in current/future CAT releases) I would prefer having the line removed and keep a leaner (readable) Dockerfile script. Thanks @octomike

Reply:
I tried the newer release of CAT (12.8.2) and this problem remains an open issue with Singularity. I would like to better understand what is going on under the curtains, why are both -a matlab script `cat_long_main.m` renamed to .txt and a compiled version of the same script- needed and swapped (replaced) when longitudinal segmentation is executed.